cmp/relativetimestamp/RelativeTimestamp.js

```diff
@@ -98,151 +97,56 @@ class LocalModel {
  * @param {string} [options.prefix] - Label preceding timestamp.
  * @param {string} [options.futureSuffix] - appended to future timestamps.
  * @param {string} [options.pastSuffix] - appended to past timestamps.
- * @param {string} [options.nowString] - string to return when timestamp is within `nowEpsilon`.
- * @param {number} [options.nowEpsilon] - threshold interval (in seconds) for `nowString`.
+ * @param {string} [options.equalString] - string to return when timestamps are within `epsilon`.
+ * @param {number} [options.epsilon] - threshold interval (in seconds) for `equalString`.
  * @param {string} [options.emptyResult] - string to return when timestamp is empty/falsey.
+ * @param {(Date|int)} [options.relativeTo] - time to which the input timestamp is compared
  */
 export function getRelativeTimestamp(timestamp, options) {
+    apiRemoved(options.nowEpsilon, 'nowEpsilon', 'Use `epsilon` instead.');
+    apiRemoved(options.nowString, 'nowString', 'Use `equalString` instead.');
+
     const defaultOptions = {
             allowFuture: false,
             short: XH.isMobile,
-            futureSuffix: 'from now',
-            pastSuffix: 'ago',
-            nowString: null,
-            nowEpsilon: 30,
-            emptyResult: ''
+            futureSuffix: options.relativeTo ? `from ${fmtCompactDate(options.relativeTo)}` : 'from now',
+            pastSuffix: options.relativeTo ? `before ${fmtCompactDate(options.relativeTo)}` : 'ago',
+            equalString: null,
+            epsilon: 30,
+            emptyResult: '',
+            relativeTo: Date.now()
         },
         opts = Object.assign({timestamp}, defaultOptions, options);
 
     if (!timestamp) return opts.emptyResult;
 
-    // Enhance options with needed info, last function will output result.
-    return flow(
-        getNow,
-        getElapsedTime,
-        normalizeAndValidate,
-        getMillisAndUnit,
-        getPrefix,
-        getSuffix,
-        getResult
-    )(opts);
+    return doFormat(opts);
 }
 
-
 //------------------------
 // Implementation
 //------------------------
-const getNow = opts => ({...opts, now: Date.now()});
+const doFormat = opts => {
```

Comment:
hoist style not to use fat arrow function for top level functions in a file.  Use `function doFormat() {}' instead.

Reply:
Done.